src/blocks/modal.css

```diff
@@ -33,6 +33,15 @@
   border-radius: 8px;
 }
 
```

Comment:
* The `avatar` modal texts should have font `Poppins` as well. Please, add `font-family: Poppins, Arial, Helvetica, sans-serif;` for the whole modal in `.modal` because all texts in the modals have the same font

In line 2 you'll have this:

```css
.modal {
  font-family: Poppins, Arial, Helvetica, sans-serif;
  position: fixed;
  top: 0;
//  ...

```